content/2_ModuleTwo_Setup/21_SelfGuidedSetup.md

```diff
@@ -0,0 +1,26 @@
+---
+title: "Self-Guided Setup Instructions" # MODIFY THIS TITLE
+chapter: true
+weight: 1 # MODIFY THIS VALUE TO REFLECT THE ORDERING OF THE MODULES
+---
+
+<!-- MORE SUBMODULES CAN BE ADDED TO DIVIDE UP THE SETUP INTO SMALLER SECTIONS -->
+<!-- COPY AND PASTE THIS SUBMODULE FILE, RENAME, AND CHANGE THE CONTENTS AS NECESSARY -->
+
```

Comment:
More content guidance ala: https://circleci.awsworkshop.io/030_self_guided_setup/30_aws_setup_your_own.html, with the exception of the warning note as it's only relevant to MP workshop series.

Most of the content should have these instructions verbatim.